operations/locustfile.py

```diff
@@ -96,3 +105,9 @@ def get_demographics_request_body():
     # read the sample request body for the demographics endpoint
     with open("e2e/src/test/resources/newborn_patient.json", "r") as f:
         return f.read()
+
+
+def get_orders_request_body():
```

Comment:
Looks good, it's just missing the instantiation of `order_request_body = get_orders_request_body()` inside `test_start`